src/main/java/graphql/execution/nextgen/ExecutionHelper.java

```diff
@@ -5,8 +5,6 @@
 import graphql.execution.ExecutionContext;
 import graphql.execution.ExecutionId;
 import graphql.execution.ExecutionStepInfo;
-import graphql.execution.FieldCollector;
-import graphql.execution.FieldCollectorParameters;
 import graphql.execution.MergedSelectionSet;
```

Comment:
Use `graphql.execution.nextgen.FieldCollector` and `graphql.execution.nextgen.FieldCollectorParameters` which keep same before this pr.